src/hyp3_isce2/burst.py

```diff
@@ -119,8 +119,7 @@ def create_burst_request_url(params: BurstParams, content_type: str) -> str:
     """
     filetypes = {'metadata': 'xml', 'geotiff': 'tiff'}
     extension = filetypes[content_type]
-    burst_number_zero_indexed = params.burst_number - 1
-    url = f'{URL}/{params.granule}/{params.swath}/{params.polarization}/{burst_number_zero_indexed}.{extension}'
+    url = f'{URL}/{params.granule}/{params.swath}/{params.polarization}/{params.burst_number}.{extension}'
```

Comment:
Important note: the top-level documentation for the burst number says that it should be 0-indexed, so I've updated this part of the code to reflect that. If we actually do want 1-indexed burst numbers, then this change should be reverted and the documentation should be changed.

Reply:
I think this is a good change. It's best if our burst numbers are indexed in the same way as the extractor.